roles/create-devstack-local-conf/tasks/main.yml

```diff
@@ -1,11 +1,44 @@
-# gophercloud
-- name: create devstack local conf with {{ os_branch }} for {{ sdk }}
+# Populate local conf header
+- name: create devstack local conf header
   shell:
     cmd: |
       set -e
       set -x
       cat << EOF >> /tmp/dg-local.conf
       [[local|localrc]]
+      EOF
```

Comment:
NETWORK_GATEWAY=10.0.0.1 should be here, make all the jobs to use the same network configuration.

Reply:
thanks for your advise, already fixed in the new RP